src/internet_identity/src/storage.rs

```diff
@@ -311,7 +338,9 @@ impl<M: Memory + Clone> Storage<M> {
             header_memory,
             anchor_memory,
             maybe_memory_manager,
-        }
+        };
+        storage.flush();
```

Comment:
Why is this flush now necessary?

Reply:
So that a new, empty storage (without any anchors) is properly initialised.

Reply:
So up to now, the `storage` was initialized only half-way and then `flush`ed later. Definitely a good change. 👍